docs/AUGUR_INT.md

```diff
@@ -0,0 +1,64 @@
+# **8Knot+Augur Integration**
+
+## **Intro**
+---
+
+A user should be able to:
+1. Create a group of repos and orgs that they want to see aggregate for and
+2. Access that group in 8Knot.
+
+## **Preparation**
+---
+1. Clone [github.com/oss-aspen/8Knot/augur_int]
```

Comment:
should this still be augur_int branch?

Reply:
yeah but tbh this whole doc should be changed so I'll do that and update.